cloudinit/distros/bsd_util.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
```

Comment:
if this is based on #62, why is this file added again?

Reply:
because the NetBSD patch is in the first in the series :-).